js/index.js

```diff
@@ -14,34 +14,52 @@ for (const item of skills){
     skillsList.appendChild(skill);
 }
 //Handle Message Form Submit
-const messageForm = document.getElementById('leave_message');
-messageForm.addEventListener("submit", function(event){    
+const messageForm = document.querySelector('form[name="leave_message"]');
+const input = messageForm.querySelector("input");
+messageForm.addEventListener("submit", (event) => {    
 
     event.preventDefault();
     
     const usersName = event.target.usersName.value;
     const usersEmail = event.target.usersEmail.value;
     const usersMessage = event.target.usersMessage.value;
     
+    const ul = document.getElementById('responds');
+    const li = document.createElement('li');
+    ul.appendChild(li);
+
     console.log('Name', usersName);
     console.log('Email', usersEmail);
     console.log('Message', usersMessage);
 
-    messageForm.reset();
- }, true);
+    const messageSection = document.getElementById('messages');
+    const messageList = messageSection.querySelector('ul');
+    const newMessage = document.createElement('li');
+    const usersLink = document.createElement('a');
+    usersLink.href = 'mailto:' + usersEmail;
+    usersLink.textContent = usersName;
 
- //Display Messages in List
-const messageSection = document.getElementById('messages');
-const messageList = messageSection.querySelector('ul');
-const newMessage = document.createElement('li');
-const removeButton = document.createElement('button');
+    const messageSpan = document.createElement('span');
+    messageSpan.textContent = usersMessage;
+
+    li.appendChild(usersLink);
+    li.appendChild(document.createTextNode(':'));
+    li.appendChild(messageSpan);
+    ul.appendChild(li);
 
-removeButton.addEventListener('click', () => {    
-    removeButton.innerText = 'remove';
-    removeButton.Type = 'button';
-    newMessage.innerHTML = `<a href="mailto:${usersEmail}">${usersName}</a><span>${usersMessage}</span>`;    
-    const entry = removeButton.parentNode;
-    entry.remove();
-    newMessage.appendChild(removeButton);
+// remove button
+    const removeButton = document.createElement('button');
+    removeButton.textContent = 'remove';
+    li.appendChild(removeButton); 
     messageList.appendChild(newMessage);
+    messageForm.reset();  
+ }, required);
+
+ //Display Messages in List
+document.getElementById('responds').addEventListener('click', (event) => {    
+   if (event.target.tagName === 'BUTTON') {
+     const li = e.target.parentNode;
+     const ul = li.parentNode;
+     ul.removeChild(li);
+   }
```

Comment:
The remove button even listener needs to be created inside the submit button event handler.